.gitignore

```diff
@@ -1,5 +1,10 @@
 # Dev stuff
 __pycache__
 *.sw?
+
+# editor stuff
+.idea
+.vscode
```

Comment:
I wonder if we actually want to keep this in order to standardise things? Eg location of Python stubs?

Reply:
Reverted back.